### PR TITLE
Ciel string title TL day 5-7,10

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

**Comment by Volvo-hackangel** [comment]
On second thought, I'll just merge this now and ask for the tlc later when string summaries are translated.

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
-霧の中
-苛立ち
-冷気の檻
-シエル先輩を見る。
-ヴローヴを見る。
-凍土のワルツ
-安息こそ故郷
-おやすみなさい。
-後処理
+The ladder to paradise
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:541]: If this is 5_1E, this is probably relating to 天使の梯子, which is translated as Jacob's Ladder. Maybe make heaven explicit, or just translate this as Jacob's Ladder as well

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+Hands of flame
+Save Noel.
+Impossible severance
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:544]: This is Shiki slicing Vlov's arm off. Amputation or something might be more apt.

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+Abandon noel.
+―――I can get out of this!
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:546]: I can keep on going!

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+The limit: a single breath...!
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:547]: I'm at my limit, I have to take a breath...!

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+The 27 Ancestors
+Rushing over to Senpai.
+Rushing over to Noel.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:550]: Try and keep active form (I rush over to {Senpai,Noel})

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+One-on-one lesson with Noel
+Escaping the depths of the Earth
+Offending Arcueid
+In the mist
+Frustration
+Cage of cold
+Looking at Ciel-senpai.
+Looking at Vlov.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:558]: Keep active / match choice text
 > I look at/toward {Ciel-senpai,Vlov}

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+Waltz on frozen soil
+In the repose of a hometown
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:560]: Tempted to tl this as an idiom - Home is where the heart is

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -538,28 +538,28 @@ I head downstairs.
...
+Goodnight.
+Post-treatment
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:562]: Cleaning up the aftermath

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -574,20 +574,20 @@ Take a walk around the courtyard.
...
-阿良句博士の普通の診察
-午後をどう過ごそう。
-中庭を散歩してみよう。
-館内を観覧してみよう。
-先輩のアパートに行ってみようか？
-休日の終わりに。
-不老不死
-ある吸血鬼の死。
+6th day - Morning
+Jump in! The Sanzu River
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:578]: 三途の川 is the equivalent of the Styx.

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -574,20 +574,20 @@ Take a walk around the courtyard.
...
+Apologize sincerely.
+Try to fool her.
+Tell her the truth.
+Having convinced Akiha...
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:582]: I've successfully convinced Akiha.

File: system_strings/sysmes_text.en [hunk 11]
```diff
@@ -574,20 +574,20 @@ Take a walk around the courtyard.
...
+A regular exam with Dr. Arach
+How should I spend my afternoon?
+Take a walk around the courtyard.
+Take a walk around the courtyard.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:586]: Should be take a walk around inside [the mansion]

File: system_strings/sysmes_text.en [hunk 12]
```diff
@@ -574,20 +574,20 @@ Take a walk around the courtyard.
...
+Go to Senpai's apartment.
+At the end of my day off.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:588]: My day off draws to a close

File: system_strings/sysmes_text.en [hunk 13]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
 Returning home at midnight.
-七日目・朝
-新しい日常へ。
-ランチタイム
-カロリーの高い定食を頼もう。
-購買でカレーパンを買い占めよう。
+7th day - Morning
+To a new life.
+Lunchtime
+Ordering the high-calorie special
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:615]: I'll order the high calorie special

File: system_strings/sysmes_text.en [hunk 14]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Buying up all the curry buns
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:616]: I'll buy up a bunch of curry buns

File: system_strings/sysmes_text.en [hunk 15]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
 お弁当のアテがあったような……？
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:617]: Doesn't someone owe me a bento...?

File: system_strings/sysmes_text.en [hunk 16]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
-放課後の茶道室
-校舎裏に行く。
-屋敷に帰ろう。
-（表向き）和やかなお茶会
-アルクェイドのお願い。
-夜回り吸血鬼
-代行者たち
-魔眼の代償
-憐憫
+That of a healthy boy
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:618]: In context this is referring to 健全な男子高校生の素直な気持ち, so maybe reify it a bit to
> Honest feelings of a healthy boy

File: system_strings/sysmes_text.en [hunk 17]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Which Executor?
+Going to the tea ceremony club
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:620]: I head to the tea ceremony clubroom

File: system_strings/sysmes_text.en [hunk 18]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Tasteless man
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:621]: _A_ tasteless man.

File: system_strings/sysmes_text.en [hunk 19]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+What is, and what can be
+Tasteful man
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:623]: _A_ tasteful man.

File: system_strings/sysmes_text.en [hunk 20]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
-アルクェイドのお願い。
-夜回り吸血鬼
-代行者たち
-魔眼の代償
-憐憫
+That of a healthy boy
+Which Executor?
+Going to the tea ceremony club
+Tasteless man
+What is, and what can be
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:622]: Might go with 'The past, and the future'

File: system_strings/sysmes_text.en [hunk 21]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Tasteful man
+After school tea ceremony room
+Going behind the school
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:625]: I head behind the school.

File: system_strings/sysmes_text.en [hunk 22]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Going home to the mansion
+A calm-seeming tea party
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:627]: An (ostensibly) peaceful tea party

File: system_strings/sysmes_text.en [hunk 23]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
+Arcueid's request.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:628]: Not checking the rest since I won't have context

File: system_strings/sysmes_text.en [hunk 24]
```diff
@@ -609,27 +609,27 @@ More details regarding vampires.
...
-憐憫
+That of a healthy boy
+Which Executor?
+Going to the tea ceremony club
+Tasteless man
+What is, and what can be
+Tasteful man
+After school tea ceremony room
+Going behind the school
+Going home to the mansion
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:626]: Time to head back to the mansion.

**Pull request merged**